config/locales/en.yml

```diff
@@ -162,8 +162,8 @@ en:
           create_account_html: or %{create_account_link}.
           description_html: Post jobs and manage applications.
           link_text:
-            create_account: create an account
-            sign_in: Sign in
+            create_account: create a hiring staff account
+            sign_in: Hiring staff sign in
```

Comment:
We've changed the 'Sign in' button for hiring staff to 'Hiring staff sign in' - could we do the same for jobseekers so theirs says 'Jobseeker sign in'?

Reply:
Could we also replicate this in the footer links, too?

Reply:
Changes done!